### PR TITLE
Register test: TC07, TC10, TC11

### PR DESCRIPTION
**Pull request by sattdn210106**
TC07-User can create a new account
TC10-User can't create an account with "Confirm password" is not the same with "Password"
TC11-User can't create an account while password and PID fields are empty

### CODE REVIEW

File: src/main/java/models/Account.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    public String getConfirmPassword() {
+        return confirmPassword;
+    }
+
+    public String getPid() {
+        return pid;
+    }
+
+    public Account getRandomAccount() {
```
Comment (tanmle) [inline src/main/java/models/Account.java]: why don't you use a constructor (public Account() {) here?

File: src/test/java/tests/RegisterTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        registerPage.register(account);
+
+        String actualGeneralErrorMsg = registerPage.getGeneralErrorMsg();
+
+        Assert.assertEquals(actualGeneralErrorMsg, expectedGeneralErrorMsg, "General error message is incorrect");
+    }
+
+    @Test(description = "User can't create account while Password and PID fields are empty")
+    public void TC011() {
+        String password = "";
```
Comment (tanmle) [inline src/test/java/tests/RegisterTest.java]: don't need to define var for a blank.

File: src/main/java/models/Account.java (hunk 1 above)
Comment (sattdn210106) [reply src/main/java/models/Account.java]: Thanks for your review! I fixed this mistake. Please review it

File: src/test/java/tests/RegisterTest.java (hunk 2 above)
Comment (sattdn210106) [reply src/test/java/tests/RegisterTest.java]: Thanks for your review! I fixed this mistake. Please review it

**Pull request merged**